### ISSUE TITLE
Minor: Interaction Cursor persists into self interaction.

### ISSUE THREAD

**Issue by J-Ziegler** [acemod/ACE3#1399]
ACE3 Version: 3.0.1 (Current build)

**Mods:**
* @cba_a3
* @ace3
* @Ares (The Zeus enhancement)

**Placed ACE3 Modules:**
* Interact
* Medical Settings
* Advanced Medical Settings
* Set Medic Class

**Description:**
If close to an AI, opening the self-interaction menu can cause the cursor to persist into the self-interaction menu even when the cursor is disabled. The menu becomes much less responsive. This is a very minor issue and very hard to reproduce, and easily resolved at a personal level.

Video:
https://www.youtube.com/watch?v=EiyQoLr3hRw
https://dl.dropboxusercontent.com/u/47686133/ACE3%20Interaction%20Cursor%20Bug.flv

**Steps to reproduce:** (Very difficult)
* Walk up to a Player/AI
* Self interact, but press the interact key (LWIN) before the self modifier (CTRL)
* Move the mouse over (or start by looking at) an interaction point on the player/ai. This must be done before the options fully open.

**Where did the issue occur?**
Editor

**RPT log file:**
http://pastebin.com/yKVBJgfy
https://dl.dropboxusercontent.com/u/47686133/Interaction%20Cursor%20Bug.rpt

**Comment by jonpas** [comment]
Duplicate of  #767
Thank you for your report though.

**Comment by esteldunedain** [comment]
I think this is obsolete, unless you can repro this with the current master and CBA R7.

**Comment by J-Ziegler** [comment]
Probably. I hadn't realized the above was the same issue. I'll look into it with the new CBA once that's out.

**Comment by esteldunedain** [comment]
It's not exactly technically the same issue, but it's related. The bug you reported happens because the menu is closed and reopened on the same frame, which should be impossible without the cba bug.
